src/Chapter2.hs

```diff
@@ -621,7 +631,12 @@ Write a function that takes elements of a list only in even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [a] -> [a]
+takeEven [] = []
+takeEven (h:t) = h: takeOdd t
+  where
+    takeOdd [] = []
+    takeOdd (_:t) = takeEven t
```

Comment:
Oh, this is interesting!

Alternatively, you could use pattern matching in `takeEven` instead. As you need to take every second element, it is possible to represent with a few pattern matching clauses 🙂